### PR TITLE
[script.module.libmediathek3] 1.5.4

### PR DESCRIPTION
**Pull request by sarbes**
### Description
Added some additional functions to my mini framework.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.libmediathek3/lib/libmediathek3dialogs.py [hunk 1]
```diff
@@ -19,10 +19,6 @@ def dialogDate():
...
 	
 def getSearchString():
-	if libmediathek3utils.searchWorkaroundExists():
-		d = libmediathek3utils.searchWorkaroundRead()
-	else:
-		dialog = xbmcgui.Dialog()
-		d = dialog.input(libmediathek3utils.getTranslation(31039),type=xbmcgui.INPUT_ALPHANUM)
-		libmediathek3utils.searchWorkaroundWrite(d)
+	dialog = xbmcgui.Dialog()
+	d = dialog.input(libmediathek3utils.getTranslation(31039),type=xbmcgui.INPUT_ALPHANUM)
```
Comment (enen92) [inline script.module.libmediathek3/lib/libmediathek3dialogs.py]: If I am not mistaken, d will return -1 if canceled. Are you taking this into account?

File: script.module.libmediathek3/lib/libmediathek3utils.py [hunk 2]
```diff
@@ -27,22 +29,18 @@ def log(msg):
...
+def _request(url,headers,post,cookies):
 	log(url)
 	if post:
-		#log(urllib.urlencode(post))
-		#log(urllib.quote(post))
-		#log(post)
-		#req = urllib2.Request(url,urllib.urlencode(post))
-		#req = urllib2.Request(url,urllib.quote(post))
 		req = urllib2.Request(url,post)
+		log('########POST!')
```
Comment (enen92) [inline script.module.libmediathek3/lib/libmediathek3utils.py]: Unnecessary log spam. Component logging for libcurl already logs any requests

File: script.module.libmediathek3/lib/libmediathek3utils.py [hunk 3]
```diff
@@ -53,7 +51,41 @@ def _request(url,headers,post):
...
+			c = pickle.loads(f_open(pathUserdata('cookies.txt')))
+			for cookie in c:
+				cj.set_cookie(cookie)
+		opener = urllib2.build_opener(urllib2.HTTPCookieProcessor(cj))
+		#response = urllib2.urlopen(req)
+		response = opener.open(req)
+		
+		c = []
+		for cookie in cj:
+			log(str(cookie))
```
Comment (enen92) [inline script.module.libmediathek3/lib/libmediathek3utils.py]: Logging cookies is a bad practice from a security pov

File: script.module.libmediathek3/lib/libmediathek3utils.py [hunk 4]
```diff
@@ -53,7 +51,41 @@ def _request(url,headers,post):
...
+			c.append(cookie)
+		log(str(cj))
```
Comment (enen92) [inline script.module.libmediathek3/lib/libmediathek3utils.py]: More spam

**Comment by enen92** [comment]
ping @sarbes

**Comment by Rechi** [comment]
The correct default English folder name is `resource.language.en_gb`, not `resource.language.en_us`.

**Comment by enen92** [comment]
@sarbes please re-submit if still interested. Closing as the PR is stalled.

**Pull request closed without merging**